### PR TITLE
docs(general): fix pre-commit link

### PR DESCRIPTION
**Pull request by tsmithv11**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

### CODE REVIEW

**Review by JBakstPaloAlto — APPROVED** [review]

File: docs/4.Integrations/pre-commit.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ nav_order: 6
 
 # Pre-Commit
```
Comment (JBakstPaloAlto) [inline docs/4.Integrations/pre-commit.md:8]: Pre-Commit Hooks

File: docs/4.Integrations/pre-commit.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ nav_order: 6
...
 
-If you want to automatically run `checkov` when files in your git repo change, [install the pre-commit binary](https://pre-commit.com/#install), and add a [.pre-commit-config.yaml file](./.pre-commit-config.yaml) to your project with content similar to the example below.
+If you want to automatically run `checkov` when files in your git repo change, [install the pre-commit binary](https://pre-commit.com/#install), and add a [.pre-commit-config.yaml file](https://github.com/bridgecrewio/checkov/blob/main/.pre-commit-config.yaml) to your project with content similar to the example below.
```
Comment (JBakstPaloAlto) [inline docs/4.Integrations/pre-commit.md:10]: To automatically run `Checkov` whenever files in your Git repository change, first [install the pre-commit binary](https://pre-commit.com/#install) and then add a [.pre-commit-config.yaml] file (https://github.com/bridgecrewio/checkov/blob/main/.pre-commit-config.yaml)) to your project with content similar to the example below.

File: docs/4.Integrations/pre-commit.md [hunk 3]
```diff
@@ -7,7 +7,7 @@ nav_order: 6
...
 
 Note that depending on the hook id you select for pre-commit hooks, you may need to provide the following:
```
Comment (JBakstPaloAlto) [inline docs/4.Integrations/pre-commit.md:12]: NOTE: Depending on the hook id you select for pre-commit hooks, you may need to provide the following:

**Review by JBakstPaloAlto — CHANGES_REQUESTED** [review]
I cannot make additional suggestions

**Review by JBakstPaloAlto — APPROVED** [review]

**Review by bo156 — APPROVED** [review]

**Pull request merged**